packages/tocco-ui/src/Button/Button.stories.js

```diff
@@ -20,7 +20,7 @@ storiesOf('Button', module)
           onClick={action('clicked')}
           look={select('look', {'-': null, ..._pick(stylingLook, ['FLAT', 'RAISED'])}) || undefined}
           iconPosition={select('iconPosition', {'-': null, ...stylingPosition}) || undefined}
-          pending={boolean('pending', false) || undefined}
+          pending={boolean('pending', false)}
```

Comment:
why did you change that? this will generate
`<Button label="My Button" onClick={clicked} pending={false} />`

but 

`<Button label="My Button" onClick={clicked}  />`
is desired